nautobot/docs/guides/graphql.md

```diff
@@ -0,0 +1,366 @@
+# GraphQL User Guide
+
+## Introduction
+
+### What is GraphQL?
+
+GraphQL is a query language for your APIs and a runtime for fulfilling those queries with your existing data.<sup>(1)</sup>
```

Comment:
IMO just replace this `sup` with an actual link to the website and ditch the "Citations" section.